docs/manifest-devhub.json

```diff
@@ -827,6 +827,12 @@
 		"markdown_source": "../packages/components/src/menu-items-choice/README.md",
 		"parent": "components"
 	},
+	{
+		"title": "StepperCell",
+		"slug": "stepper-cell",
+		"markdown_source": "../packages/components/src/mobile/bottom-sheet/stepper-cell/README.md",
+		"parent": "components"
+	},
```

Comment:
Not sure if we should add this here 🤔 
`StepperCell` is an internal component that we make available via the native version of `StepperControl`.

@gziolo any light on this?

Reply:
It's interesting that it gets detected. I guess this is how it works with `README.md` files added to `@wordpress/components`.

I don't think it's internal as it gets exposed and consumed in the block library. We should just make it clear that it will work only with the React Native code. @youknowriad - any thoughts on how to make it possible to ignore all mobile docs from being exposed in the documentation published at https://developers.wordpress.org?

Reply:
> I don't think it's internal as it gets exposed and consumed in the block library.

We are currently using the controls abstractions (StepperControl, RangeControl, etc...), so the mobile-only `-Cell` components are not being consumed by any other libraries.

We are planning on adding these docs directly to StepperControl instead. Wdyt?